src/network/networkd-link.c

```diff
@@ -721,10 +721,15 @@ void link_check_ready(Link *link) {
                     !link->ipv4ll_route)
                         return;
 
-        if (link_ipv6ll_enabled(link))
+        if (link_ipv6_enabled(link)) {
+
                 if (in_addr_is_null(AF_INET6, (const union in_addr_union*) &link->ipv6ll_address) > 0)
                         return;
 
+                if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured)
```

Comment:
That wasn't the confusion actually -- with bridges you can have it both ways, either you treat the bridge itself as "the" device and assign IPs etc. to it and the components (slaves) don't have addresses on their own (although that then acts more like a bond) -- or you can have an "anonymous" bridge with no addresses on its own, but where its individual components have addresses (the more "classic" usage of a bridge). Like this it looks like it would completely break the second case?